.github/workflows/pull-request.yaml

```diff
@@ -5,6 +5,7 @@ on: pull_request
 # https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
 permissions:
   checks: write
+  contents: write
```

Comment:
I didn't realize token permissions are finally a thing, sick!

Reply:
Yup. `GITHUB_TOKEN` has everything by default, and `permissions` gives only the ones we define. We can now get rid of `PAT_DECRAPIFIER` if we want to. Another cool thing is that we can give permissions to the token at the job level instead of the workflow level: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions